connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -268,15 +295,16 @@ object KustoWriter {
       ingestClient: IngestClient,
       partitionsResults: CollectionAccumulator[PartitionResult],
       batchIdForTracing: String,
+      ingestionProperties: IngestionProperties, // TODO not used
```

Comment:
so use that ? instead of calculating the properties again ..

Reply:
Removed this unused parameter

Reply:
I think you should pass it and use it - why call getIngestionProperties again ...